<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>rpms/telepathy-qt.git - telepathy-qt</title>
<meta name='generator' content='cgit v0.12'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/rpms/telepathy-qt.git/atom/0021-Farstream-gst-gstconfig.h-can-be-in-LIBDIR-search-fo.patch?h=master' type='application/atom+xml'/>
<link rel='vcs-git' href='git://pkgs.fedoraproject.org/rpms/telepathy-qt.git' title='rpms/telepathy-qt.git Git repository'/>
<link rel='vcs-git' href='ssh://pkgs.fedoraproject.org/rpms/telepathy-qt.git' title='rpms/telepathy-qt.git Git repository'/>
<link rel='vcs-git' href='http://pkgs.fedoraproject.org/git/rpms/telepathy-qt.git' title='rpms/telepathy-qt.git Git repository'/>
</head>
<body>
<script src="https://apps.fedoraproject.org/fedmenu/js/jquery-1.11.2.min.js"></script>
<script src="https://apps.fedoraproject.org/fedmenu/js/fedmenu.js"></script>

<script>
    var base = 'https://apps.fedoraproject.org/';
    var tokens = window.location.href.split('/');
  var repo = null;
  if (tokens.length > 4 && tokens[4] != '') {
    repo = tokens[4].split('.')[0];
  }
  fedmenu({
      'url': base + 'js/data.js',
      'mimeType': 'application/javascript',
      'position': 'bottom-right',
      'package': repo,
  });
</script>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='rpms/telepathy-qt.git' href='/cgit/rpms/telepathy-qt.git/'>rpms/telepathy-qt.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='f21'>f21</option>
<option value='f22'>f22</option>
<option value='f23'>f23</option>
<option value='f24'>f24</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>telepathy-qt</td><td class='sub right'>Jon Ciesla</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/rpms/telepathy-qt.git/'>summary</a><a href='/cgit/rpms/telepathy-qt.git/refs/'>refs</a><a href='/cgit/rpms/telepathy-qt.git/log/0021-Farstream-gst-gstconfig.h-can-be-in-LIBDIR-search-fo.patch'>log</a><a class='active' href='/cgit/rpms/telepathy-qt.git/tree/0021-Farstream-gst-gstconfig.h-can-be-in-LIBDIR-search-fo.patch'>tree</a><a href='/cgit/rpms/telepathy-qt.git/commit/0021-Farstream-gst-gstconfig.h-can-be-in-LIBDIR-search-fo.patch'>commit</a><a href='/cgit/rpms/telepathy-qt.git/diff/0021-Farstream-gst-gstconfig.h-can-be-in-LIBDIR-search-fo.patch'>diff</a><a href='/cgit/rpms/telepathy-qt.git/stats/0021-Farstream-gst-gstconfig.h-can-be-in-LIBDIR-search-fo.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/rpms/telepathy-qt.git/log/0021-Farstream-gst-gstconfig.h-can-be-in-LIBDIR-search-fo.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/rpms/telepathy-qt.git/tree/'>root</a>/<a href='/cgit/rpms/telepathy-qt.git/tree/0021-Farstream-gst-gstconfig.h-can-be-in-LIBDIR-search-fo.patch'>0021-Farstream-gst-gstconfig.h-can-be-in-LIBDIR-search-fo.patch</a></div><div class='content'>blob: 6685ebbea4446be9147995906294c0fca7d7e969 (<a href='/cgit/rpms/telepathy-qt.git/plain/0021-Farstream-gst-gstconfig.h-can-be-in-LIBDIR-search-fo.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
<a id='n34' href='#n34'>34</a>
<a id='n35' href='#n35'>35</a>
<a id='n36' href='#n36'>36</a>
<a id='n37' href='#n37'>37</a>
<a id='n38' href='#n38'>38</a>
<a id='n39' href='#n39'>39</a>
<a id='n40' href='#n40'>40</a>
<a id='n41' href='#n41'>41</a>
<a id='n42' href='#n42'>42</a>
<a id='n43' href='#n43'>43</a>
<a id='n44' href='#n44'>44</a>
<a id='n45' href='#n45'>45</a>
<a id='n46' href='#n46'>46</a>
<a id='n47' href='#n47'>47</a>
<a id='n48' href='#n48'>48</a>
<a id='n49' href='#n49'>49</a>
<a id='n50' href='#n50'>50</a>
<a id='n51' href='#n51'>51</a>
<a id='n52' href='#n52'>52</a>
<a id='n53' href='#n53'>53</a>
<a id='n54' href='#n54'>54</a>
<a id='n55' href='#n55'>55</a>
<a id='n56' href='#n56'>56</a>
<a id='n57' href='#n57'>57</a>
<a id='n58' href='#n58'>58</a>
<a id='n59' href='#n59'>59</a>
<a id='n60' href='#n60'>60</a>
<a id='n61' href='#n61'>61</a>
<a id='n62' href='#n62'>62</a>
<a id='n63' href='#n63'>63</a>
<a id='n64' href='#n64'>64</a>
<a id='n65' href='#n65'>65</a>
<a id='n66' href='#n66'>66</a>
<a id='n67' href='#n67'>67</a>
<a id='n68' href='#n68'>68</a>
<a id='n69' href='#n69'>69</a>
<a id='n70' href='#n70'>70</a>
<a id='n71' href='#n71'>71</a>
<a id='n72' href='#n72'>72</a>
<a id='n73' href='#n73'>73</a>
<a id='n74' href='#n74'>74</a>
<a id='n75' href='#n75'>75</a>
<a id='n76' href='#n76'>76</a>
<a id='n77' href='#n77'>77</a>
<a id='n78' href='#n78'>78</a>
<a id='n79' href='#n79'>79</a>
<a id='n80' href='#n80'>80</a>
<a id='n81' href='#n81'>81</a>
<a id='n82' href='#n82'>82</a>
<a id='n83' href='#n83'>83</a>
<a id='n84' href='#n84'>84</a>
<a id='n85' href='#n85'>85</a>
<a id='n86' href='#n86'>86</a>
<a id='n87' href='#n87'>87</a>
<a id='n88' href='#n88'>88</a>
</pre></td>
<td class='lines'><pre><code>From ec4a3d62b68a57254515f01fc5ea3325ffb1dbfb Mon Sep 17 00:00:00 2001
From: Niels Ole Salscheider &lt;niels_ole@salscheider-online.de&gt;
Date: Tue, 29 Sep 2015 20:52:15 +0200
Subject: [PATCH 21/22] Farstream: gst/gstconfig.h can be in LIBDIR, search for
 it explicitly

Since GStreamer-1.5.1 config header file moved to LIBDIR.

Reviewed-by: Alexandr Akulich &lt;akulichalexander@gmail.com&gt;
---
 TelepathyQt/Farstream/CMakeLists.txt |  2 +-
 cmake/modules/FindGStreamer.cmake    | 25 ++++++++++++++++++++-----
 2 files changed, 21 insertions(+), 6 deletions(-)

diff --git a/TelepathyQt/Farstream/CMakeLists.txt b/TelepathyQt/Farstream/CMakeLists.txt
index 250bfdb..dd71430 100644
--- a/TelepathyQt/Farstream/CMakeLists.txt
+++ b/TelepathyQt/Farstream/CMakeLists.txt
@@ -2,7 +2,7 @@ if(FARSTREAM_COMPONENTS_FOUND)
     include_directories(${TELEPATHY_FARSTREAM_INCLUDE_DIR}
                         ${TELEPATHY_GLIB_INCLUDE_DIR}
                         ${FARSTREAM_INCLUDE_DIR}
-                        ${GSTREAMER_INCLUDE_DIR}
+                        ${GSTREAMER_INCLUDE_DIRS}
                         ${GLIB2_INCLUDE_DIR}
                         ${LIBXML2_INCLUDE_DIR}
                         ${DBUS_INCLUDE_DIR})
diff --git a/cmake/modules/FindGStreamer.cmake b/cmake/modules/FindGStreamer.cmake
index 63fb41c..ffdd7a9 100644
--- a/cmake/modules/FindGStreamer.cmake
+++ b/cmake/modules/FindGStreamer.cmake
@@ -13,12 +13,12 @@
 
 # TODO: Other versions --&gt; GSTREAMER_X_Y_FOUND (Example: GSTREAMER_0_8_FOUND and GSTREAMER_0_10_FOUND etc)
 
-IF (GSTREAMER_INCLUDE_DIR AND GSTREAMER_LIBRARIES AND GSTREAMER_BASE_LIBRARY AND GSTREAMER_INTERFACE_LIBRARY)
+IF (GSTREAMER_INCLUDE_DIRS AND GSTREAMER_LIBRARIES AND GSTREAMER_BASE_LIBRARY AND GSTREAMER_INTERFACE_LIBRARY)
    # in cache already
    SET(GSTREAMER_FIND_QUIETLY TRUE)
-ELSE (GSTREAMER_INCLUDE_DIR AND GSTREAMER_LIBRARIES AND GSTREAMER_BASE_LIBRARY AND GSTREAMER_INTERFACE_LIBRARY)
+ELSE (GSTREAMER_INCLUDE_DIRS AND GSTREAMER_LIBRARIES AND GSTREAMER_BASE_LIBRARY AND GSTREAMER_INTERFACE_LIBRARY)
    SET(GSTREAMER_FIND_QUIETLY FALSE)
-ENDIF (GSTREAMER_INCLUDE_DIR AND GSTREAMER_LIBRARIES AND GSTREAMER_BASE_LIBRARY AND GSTREAMER_INTERFACE_LIBRARY)
+ENDIF (GSTREAMER_INCLUDE_DIRS AND GSTREAMER_LIBRARIES AND GSTREAMER_BASE_LIBRARY AND GSTREAMER_INTERFACE_LIBRARY)
 
 IF (NOT WIN32)
    # use pkg-config to get the directories and then use these values
@@ -38,6 +38,15 @@ FIND_PATH(GSTREAMER_INCLUDE_DIR gst/gst.h
    PATH_SUFFIXES gstreamer-1.0
    )
 
+FIND_PATH(GSTREAMER_CONF_INCLUDE_DIR gst/gstconfig.h
+   PATHS
+   ${PC_GSTREAMER_INCLUDEDIR}
+   ${PC_GSTREAMER_INCLUDE_DIRS}
+   PATH_SUFFIXES gstreamer-1.0
+   )
+
+SET(GSTREAMER_INCLUDE_DIRS ${GSTREAMER_INCLUDE_DIR} ${GSTREAMER_CONF_INCLUDE_DIR})
+
 FIND_LIBRARY(GSTREAMER_LIBRARIES NAMES gstreamer-1.0
    PATHS
    ${PC_GSTREAMER_LIBDIR}
@@ -57,6 +66,12 @@ ELSE (GSTREAMER_INCLUDE_DIR)
    MESSAGE(STATUS "GStreamer: WARNING: include dir not found")
 ENDIF (GSTREAMER_INCLUDE_DIR)
 
+IF (GSTREAMER_CONF_INCLUDE_DIR)
+	#MESSAGE(STATUS "DEBUG: Found GStreamer config include dir: ${GSTREAMER_CONF_INCLUDE_DIR}")
+ELSE (GSTREAMER_CONF_INCLUDE_DIR)
+   MESSAGE(STATUS "GStreamer: WARNING: config include dir not found")
+ENDIF (GSTREAMER_CONF_INCLUDE_DIR)
+
 IF (GSTREAMER_LIBRARIES)
    #MESSAGE(STATUS "DEBUG: Found GStreamer library: ${GSTREAMER_LIBRARIES}")
 ELSE (GSTREAMER_LIBRARIES)
@@ -65,6 +80,6 @@ ENDIF (GSTREAMER_LIBRARIES)
 
 
 INCLUDE(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(GStreamer  DEFAULT_MSG  GSTREAMER_LIBRARIES GSTREAMER_INCLUDE_DIR GSTREAMER_BASE_LIBRARY)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(GStreamer  DEFAULT_MSG  GSTREAMER_LIBRARIES GSTREAMER_INCLUDE_DIRS GSTREAMER_BASE_LIBRARY)
 
-MARK_AS_ADVANCED(GSTREAMER_INCLUDE_DIR GSTREAMER_LIBRARIES GSTREAMER_BASE_LIBRARY)
+MARK_AS_ADVANCED(GSTREAMER_INCLUDE_DIRS GSTREAMER_LIBRARIES GSTREAMER_BASE_LIBRARY)
-- 
2.5.0

</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated by <a href='http://git.zx2c4.com/cgit/about/'>cgit v0.12</a> at 2016-03-19 11:23:37 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
